spk/sonarr/Makefile

```diff
@@ -0,0 +1,57 @@
+SPK_NAME = sonarr
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = 1
+SPK_ICON = src/sonarr.png
+
+DEPENDS = cross/sonarr cross/libstdc++ cross/sqlite
+
+# Arch exclusions for dotnet 6.0
+DOTNET_CORE_ARCHS = 1
+
+MAINTAINER = SynoCommunity
+MAINTAINER_URL = https://synocommunity.com/
+DESCRIPTION = Sonarr is a PVR for newsgroup and torrent users. It can monitor multiple RSS feeds for new episodes of your favourite shows and will grab, sorts and rename them. It can also be configured to automatically upgrade the quality of files already downloaded if a better quality format becomes available.
+DESCRIPTION_FRE = Sonarr est un PVR pour les utilisateurs de groupes de discussion et torrents. Il peut surveiller plusieurs flux RSS pour les nouveaux épisodes de vos séries préférées et saisira, sortes et les renomme. Il peut également être configuré pour mettre à jour automatiquement la qualité des fichiers déjà téléchargés si un meilleur format de qualité devient disponible.
+DESCRIPTION_SPN = Sonarr es un PVR para los usuarios de grupos de noticias y torrents. Se puede controlar múltiples canales RSS para nuevos episodios de sus programas favoritos y se agarra, tipo y les cambia el nombre. También puede ser configurado para actualizar automáticamente la calidad de los archivos ya descargados si un formato de mejor calidad disponible.
+CHANGELOG = "Initial Sonarr v4 package with version 4.0.0.394. This package replaces a former Sonarr v3 installation (internally named nzbdrone)."
+DISPLAY_NAME = Sonarr v4
+HOMEPAGE = https://sonarr.tv
+LICENSE  = GPLv3
+LICENSE_FILE = $(WORK_DIR)/Sonarr/LICENSE.md
+HELPURL = https://wiki.servarr.com/sonarr/troubleshooting
+SUPPORT_URL = https://wiki.servarr.com/sonarr/faq-v4
+
+STARTABLE = yes
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
+SERVICE_PORT = 8989
+SERVICE_PORT_TITLE = Sonarr Web UI
+
+# Admin link for in DSM UI
+ADMIN_PORT = $(SERVICE_PORT)
+
+WIZARDS_DIR = src/wizard/
+
+POST_STRIP_TARGET = sonarr_extra_install
+
+# use alternate TMPDIR as /tmp might be too small and not accessible on DSM >= 7.1.
+# TMPDIR is used for sonarr built in backup and update
+USE_ALTERNATE_TMPDIR = 1
+
+# replace previous legacy package
+INSTALL_REPLACE_PACKAGES = "nzbdrone>=20210329-19"
+USE_DEPRECATED_REPLACE_MECHANISM = yes
+# disable checking for port conflicts
+CHECKPORT = no
```

Comment:
Shouldn't there also be a BETA flag somewhere in there?

Reply:
I've added the beta flag as proposed.

Reply:
IMHO the BETA flag is to mark spk packages as beta and to the included package.

There was a discussion somewhere, and if I remember well, sonarr v4 will be beta for several years from now, but everyone that uses v4 will be aware of.

So if this package is proved to work, I would not mark it as beta.

Reply:
Thanks for the reminder. Yes, other packages like Lidarr and Prowlarr had operated as pre-release versions for some time without this flag so I'll remove it once the other comments are in.